core/runtime/src/main/java/io/quarkus/runtime/annotations/RunOnWorkerThread.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.runtime.annotations;
+
+import static java.lang.annotation.ElementType.*;
```

Comment:
Please remove the star import :)

Reply:
The file is going to be renamed, it will go away.